apps/arena/lib/arena/socket_handler.ex

```diff
@@ -32,6 +32,11 @@ defmodule Arena.SocketHandler do
   end
 
   @impl true
+  def websocket_handle({:text, "leave_lobby"}, state) do
```

Comment:
I would like opinions on this, receive the action as a plain text rather than protobuf. On one hand this is waaayy simpler and works, on the other it kinda goes against having everything in protobuf

So looking for opinions

Reply:
i don't see it as a big issue, i think protobuf as a way to transport structs between the different places, a leave lobby signal is not a struct so i don't see it as an issue